Makefile

```diff
@@ -282,13 +282,28 @@ go_imports: check_go_version ## Run goimports on all go files
 ### Tests ###
 #############
 
-.PHONY: test_e2e
-test_e2e: acc_initialize_pubkeys_warn_message ## Run all E2E tests
+.PHONY: test_e2e_env
+test_e2e_env: acc_initialize_pubkeys_warn_message ## Setup the default env vars for E2E tests
 	export POCKET_NODE=$(POCKET_NODE) && \
 	export APPGATE_SERVER=$(APPGATE_SERVER) && \
-	POKTROLLD_HOME=../../$(POKTROLLD_HOME) && \
+	export POKTROLLD_HOME=../../$(POKTROLLD_HOME)
+
+.PHONY: test_e2e
+test_e2e: test_e2e_env ## Run all E2E tests
 	go test -v ./e2e/tests/... -tags=e2e,test
 
+.PHONY: test_e2e_session
+test_e2e_session: test_e2e_env ## Run only the E2E suite that exercises the session (i.e. claim/proof) life-cycle
+	go test -v ./e2e/tests/... -tags=e2e,test --features-path=session.feature
+
+.PHONY: test_e2e_settlement
+test_e2e_settlement: test_e2e_env ## Run only the E2E suite that exercises the session & tokenomics settlement
+	go test -v ./e2e/tests/... -tags=e2e,test --features-path=0_settlement.feature
+
+.PHONY: test_load_relays_stress
```

Comment:
I don't think this exists yet. Assuming it'll come soon though?

Not going to block.

Reply:
Yes, it's in #432.